tests/sles4sap/sap_suse_cluster_connector.pm

```diff
@@ -79,6 +79,7 @@ sub run {
     my $rsc = get_var('NW') ? "rsc_sap_${instance_sid}_${instance_type}${instance_id}" : "rsc_SAPHana_${instance_sid}_${instance_type}${instance_id}";
     exec_conn_cmd(binary => $binary, cmd => "$_ --res $rsc --act stop", timeout => 120) foreach qw(fra cpa);
     sleep 10;      # wait to let enough time for the HA stack to stop the resource
```

Comment:
Is this `sleep` command still useful now?

Reply:
> Is this `sleep` command still useful now?

Sadly, yes. Just saw a test fail in pvm_hmc backend with root-ssh-serial terminal because `wait_until_resources_stopped` ran before the resource status changed to *Stopping*. So the loop in `wait_until_resources_stopped` cycled once, and then `crm_mon` showed the resource in *Stopping* status, which is precisely what we want to avoid.

Reply:
OK, understood! Maybe just adding a warning to not remove this `sleep`, mainly for me to not try to remove it in 6 months :D

Reply:
Will do.

Reply:
Removed the sleep call, but added a minimum number of times to check in `wait_until_resources_stopped` which hopefully should help us avoid this race condition.